libraries/AP_BattMonitor/AP_BattMonitor_ESC.cpp

```diff
@@ -20,6 +20,33 @@
 
 #include "AP_BattMonitor_ESC.h"
 
+const AP_Param::GroupInfo AP_BattMonitor_ESC::var_info[] = {
+
+    // Param indexes must be between 36 and 39 to avoid conflict with other battery monitor param tables loaded by pointer
+
+    // @Param: ESC_MASK
+    // @DisplayName: ESC mask
+    // @Description: 0: Use all connected ESC's if none zero use those set in mask
```

Comment:
This 0: at the beginning is weird, and the rest doesn't quite make sense. Maybe add a period or a comma somewhere?

Reply:
I have re-worded .